packages/exists/src/existsModule.ts

```diff
@@ -0,0 +1,46 @@
+
+import { IConstraintModule, ConstraintValue, ConstraintExecuter, IRuntimeContext, ValidationError, IValidator, ConstraintConfiguration, ValidationResult } from '@configurable-validator/core'
+
+export class ExistsModule implements IConstraintModule {
+
+    initialize(validator: IValidator) {
+    }
+
+    buildConstraintExecuter(value: ConstraintValue, externalData?: any): ConstraintExecuter {
+        return (data: object, context: IRuntimeContext) : ValidationResult => {
+            let exists: boolean;
+            let validationError: ValidationError;
+            if(typeof value === "boolean") {
+                exists = (data != undefined) == value;
```

Comment:
this should be `data !== undefined`. maybe you meant `data != null` which is nill check?

Reply:
exists means that the property exists on the object which really means I should use `data !== undefined`